integration_tests/configs/default.jsonnet

```diff
@@ -3,7 +3,13 @@
   'cronos_777-1': {
     cmd: 'cronosd',
     'start-flags': '--trace',
+    config: {
+      mempool: {
+        version: 'v1',
```

Comment:
so the current default mempool settings will be using `v1?`

Reply:
yeah, `v1` support prioritization.